content/en/docs/concepts/storage/_index.md

```diff
@@ -1,5 +1,9 @@
 ---
 title: "Storage"
 weight: 70
+description: >
+  Kubernetes provides both long-term and ephemeral storage that you can make available to Pods.
```

Comment:
nit: there are several `you can` phrases in the paragraph.
A quick edit:
Kubernetes provides both long-term and ephemeral storage. Pods interface with the 
storage resources to persist and maintain the state of applications.
Stateful applications in your cluster rely upon durable storage. You can also configure
different classes of storage and automate how these are provisioned.

Reply:
Please shorten. Possible edit:
Kubernetes provides both long-term and ephemeral storage.